### PR TITLE
lock wallet after app was minimized and opened again VLWA-1648

### PR DESCRIPTION
**Pull request by Apindes**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

[VLWA-1648](https://velasnetwork.atlassian.net/browse/VLWA-1648) – Lock wallet after wallet is in the background mode (minimize the application)

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -57,6 +69,31 @@ class App extends React.Component {
...
+            this.setState({ AppReady: null });
+            AppState.removeEventListener('change', _appStateListener);
+          }
+          if (!AppReady && nextAppState === 'active') {
+            console.log('RUN 🔥');
+            this.startApp();
+          }
+        }
+      );
+      console.log();
```
Comment (FrozenPyrozen) [inline App.js:90]: Do we need logs here?

**Review by FrozenPyrozen — APPROVED** [review]

**Review by FrozenPyrozen — CHANGES_REQUESTED** [review]
Remove logs if they don’t needed

File: App.js [hunk 2]
```diff
@@ -57,6 +69,31 @@ class App extends React.Component {
...
+  componentDidMount() {
+    const { AppReady, appStateListener } = this.state;
+
+    if (!appStateListener) {
+      const _appStateListener = AppState.addEventListener(
+        'change',
+        (nextAppState) => {
+          console.log('Next AppState is= ', nextAppState);
+          if (nextAppState === 'background') {
+            this.setState({ AppReady: null });
```
[the remark this answers is not in the document]
Comment (Apindes) [reply App.js:81]: not really, AppReady is an object

File: App.js (hunk 1 above)
Comment (Apindes) [reply App.js:90]: it separates logs

**Review by FrozenPyrozen — APPROVED** [review]

**Pull request closed without merging**